lib/server.go

```diff
@@ -373,7 +373,12 @@ func (s *Server) AddData(new []SendableData) {
 	curDiff := append(added, removed...)
 	s.sdlock.RUnlock()
 
-	s.AddSDsDiff(curDiff)
+	if (len(curDiff) == 0) {
```

Comment:
```suggestion
	if len(curDiff) == 0 {
```

I think gofmt would have applied this automatically